From 675f4295379874818ea2f2b13e64ee6b199d11dd Mon Sep 17 00:00:00 2001
From: Jiang Kun <kun.jiang@intel.com>
Date: Fri, 20 Apr 2018 01:50:20 +0800
Subject: [PATCH 0976/1240] platform: xilinx: Passing media_pad as argument

Change media_pad as argument for media_graph_walk_start to
fix compilation error.

Signed-off-by: Jiang Kun <kun.jiang@intel.com>
---
 drivers/media/platform/xilinx/xilinx-dma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-dma.c b/drivers/media/platform/xilinx/xilinx-dma.c
index 522cdfd..cad9c24 100644
--- a/drivers/media/platform/xilinx/xilinx-dma.c
+++ b/drivers/media/platform/xilinx/xilinx-dma.c
@@ -193,7 +193,7 @@ static int xvip_pipeline_validate(struct xvip_pipeline *pipe,
 		return ret;
 	}
 
-	media_graph_walk_start(&graph, entity);
+	media_graph_walk_start(&graph, &entity->pads[0]);
 
 	while ((entity = media_graph_walk_next(&graph))) {
 		struct xvip_dma *dma;
-- 
2.7.4

